roles/blackbox_exporter/defaults/main.yml

```diff
@@ -71,7 +71,10 @@ blackbox_exporter_configuration_modules:
 #      validate_answer_rrs:
 #        fail_if_matches_regexp: [test]
 
-# Where to put the blackbox_exporter.yml main configuration file
-blackbox_exporter_config_dir: /etc
+# Where to put the blackbox.yml main configuration file
+blackbox_exporter_config_dir: /etc/prometheus
 
 blackbox_exporter_binary_install_dir: "/usr/local/bin"
+blackbox_exporter_binary_name: "blackbox_exporter"
```

Comment:
I don't think we want to do this.

Reply:
could you expand on this? what exactly do we want to not do here, and why?

Reply:
I don't see any need/benefit with having the binary name customizable, we should keep the options at minimal and avoid having options just for options sake.

Reply:
this is not options option's sake: i have modified this option for it to be compatible with the debian package....

Reply:
Ah apologies. At the moment we do not support other installation methods than the binary install method that's currently in the roles.
When and if we expand that into rpm/deb/etc. packages we would definitely want the roles to install those packages, then variables such as these would most likely be populated internally in the role and not user configurable.
With that being said, such a feature would be a major change and I would first like to take care of some streamlining/deduplicating of tasks in the roles we currently have. (#64 )

Reply:
ah. well, i guess one thing i'm saying is you're almost there and you just need to use the right binary name here. i've deployed prom on a debian 12 host using this playbook and it kind of worked after the bunch of patches i had to roll here...